cellmlmanip/model.py

```diff
@@ -458,6 +458,12 @@ def graph(self):
                     graph.add_node(rhs, equation=sympy.Eq(rhs, dummy), variable_type='parameter')
                     graph.add_edge(rhs, lhs)
 
+            # check that the free variable if defined as a node
```

Comment:
Could we move this up to around line 428?

Reply:
Might get added twice - since it may occur on rhs of another equation and get added there

Reply:
But you could do the same check as here, no?

Reply:
oh ok get it now